README.md

```diff
@@ -29,50 +29,25 @@ Clone the respository and ensure you have the requirements below.
  ```
 
 
-
 ## Configuration Setup
 
-1. Copy the values from the `firebaseConfig` variable in the web app in the firebase console to the `FirebaseConfigs` variable in `./config/app.js`.
-2. Download the `GoogleService-Info.plist` file from the ios app on firebase. Place this file in `./mobile/configs/app`.
-3. Copy the `GOOGLE_APP_ID` and `API_KEY` from the plist file you just downloaded into the ios object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `CLIENT_ID` from the same plist file to line 48 in the `GoogleConfigs` variable in `./config/app.js`.
-4. Copy the `REVERSED_CLIENT_ID` from the same plist file into line 39 in `./mobile/app.json`.
-5. Download the `google-services.json` file from the android app on firebase. Place this file in `./mobile/configs/app` and in `./mobile/android/app`.
-6. Copy the `current_key` and `mobilesdk_app_id` from the json file you just downloaded into the android object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `client_id` on line 18 to line 47 of the same `app.js` file.
-7. Again from the downloaded json file, copy the `current_key` and the `certificate_hash` to lines 54 and 55 of `./mobile/app.json`.
-8. Create `.runtimeconfig.json` in `./server/functions` with the following:
-
-	```
-	{
-	  "envs": {
-	    "project_name": "[appName]",
-	    "sendgrid_api_key": "...",
-	    "sendgrid_emails_validation_api_key": "...",
-	    "is_prod": "false", // set false for staging,
-	    "devlogin": true,
-	    "twilio_phone_number_from": "...",
-	    "twilio_auth_token": "...",
-	    "twilio_account_sid": "..."
-	  }
-	}
-	```
-9. Add the Sendgrid API key to the `.runtimeconfig.json` file you just created (get this key from a dev on the team).
-10. Now create `.env` in `./server/functions` with the following:
+1. Download the `GoogleService-Info.plist` file from the ios app on firebase. Place this file in `./mobile/configs/app`.
+2. Download the `google-services.json` file from the android app on firebase. Place this file in `./mobile/configs/app` and in `./mobile/android/app`.
+3. Copy the `.env-starter` file to `.env` in the root directory, and fill in each variable with the correct values. Alternatively, obtain an environment file from another developer on the team.
+4. From the root directory, run `yarn all` and wait for node modules to install.
```

Comment:
"yarn all" still errors out because the persona and react-native-switch-pro dependencies haven't been cloned yet (cloned in setup script). Just installing the envsub module first then following the rest of the flow allows the setup to work.

Reply:
Riiight yeah good point

Reply:
@cara-wong Please see the changes I made to script and readme. I tested this new workflow with a fresh clone and it works.